### PR TITLE
Add 'is-focused' class to main div, when the Input component is focused.

### PR DESCRIPTION
**Pull request by shoetten**
The app i'm working on requires styling based on whether input elements are focused or not and i think other users could benefit from this as well.
Since the css :focus selector does not apply to parent elements, this needs to be done in javascript.

### CODE REVIEW

File: lib/ReactTags.js [hunk 1]
```diff
@@ -20,7 +20,8 @@ const DefaultClassNames = {
     tagName: 'ReactTags__tagName',
     suggestions: 'ReactTags__suggestions',
     isActive: 'is-active',
-    isDisabled: 'is-disabled'
+    isDisabled: 'is-disabled',
+    isFocused: 'focused'
```
Comment (i-like-robots) [inline lib/ReactTags.js]: for consistency I think this should be `is-focused`

**Comment by i-like-robots** [comment]
Seems like a reasonable feature to me! I'm happy to add some extra styles to the demo, too.

**Comment by shoetten** [comment]
Hey, i changed the default to `is-focused` and rebased.

**Comment by i-like-robots** [comment]
Closing in favour of https://github.com/i-like-robots/react-tags/pull/34

Thanks @shoetten for your hard work on this!

**Pull request closed without merging**